modes/parkour/files.lua

```diff
@@ -119,7 +130,7 @@ onEvent("PlayerDataLoaded", function(player, data)
 
 	if killing[player] and data.parkour then
 		webhooks._count = webhooks._count + 1
-		webhooks[webhooks._count] = "**`[BANS]:`** `" .. room.name .. "` (remotely) `" .. killing[player][1] .. "`: `!kill " .. player .. " " .. killing[player][2] .. "`"
+		webhooks[webhooks._count] = "**`[KILL]:`** `" .. room.name .. "` (remotely) `" .. killing[player][1] .. "`: `!kill " .. player .. " " .. killing[player][2] .. "`"
```

Comment:
This won't get caught by the bot if it has the prefix KILL, making it show up in game-logs.